pxl_scripts/sotw/ndots/data.pxl

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' Kubernetes ndots demo
+'''
+
+import px
+
+# Prep
+# kubectl apply -f https://k8s.io/examples/admin/dns/dnsutils.yaml
+
+# Then trigger a DNS request
+# kubectl exec -i -t dnsutils -- nslookup news.ycombinator.com
+
+# ----------------------------------------------------------------
+# Script variables
+# ----------------------------------------------------------------
+max_num_records = 1000
```

Comment:
We should probably pull out variables like this to be script argument.

In this case, we can also just completely remove this and the corresponding `df.head` below.